ts/state/ducks/SessionTheme.tsx

```diff
@@ -331,13 +331,20 @@ export const SessionGlobalStyles = createGlobalStyle`
     --margins-md:  15px;
     --margins-lg:  20px;
 
+    /* PADDING */
+    --padding-message-content: 7px 13px;
+    --padding-link-preview: -7px -13px 7px -13px; // bottom has positive value because a link preview has always a body below
```

Comment:
Can you add a // TODO Theming here I will need to rename these to match the new system. 🙏